src/asm_cfg.cpp

```diff
@@ -138,6 +144,8 @@ static cfg_t to_nondet(const cfg_t& cfg) {
     return res;
 }
 
+/// Get the type of a given instruction.
+/// Most of these type names are also statistics header labels.
 static std::string instype(Instruction ins) {
```

Comment:
It would be better to rename - `name_of_instruction_category()` or something like that.

Reply:
Agree renames of lots of things would help.  But I was thinking to do only comments in this PR.  Let me know.

Reply:
The one thing I did do in the update to this PR though is to add an enum for register numbers to reduce hard coding of magic numbers.  I think it makes it much more readable, let me know.